proto/decentraland/social/friendships/friendships.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+package decentraland.social.friendships;
+
+import "google/protobuf/empty.proto";
```

Comment:
Is there any benefit in using this instead of a custom Empty?

Reply:
Why implement a custom one when there's already one available?